README.md

```diff
@@ -1,2 +1,22 @@
 # guidebook
```

Comment:
# Guidebook

This repository contains source materials The Open Source Way 2.0, a (somewhat opinionated) guidebook for anyone interested in managing open source communities. It collects best practices for initiating, nurturing, growing, and maintaining groups of passionate contributors.

The Open Source Way 2.0 refreshes [pre-existing materials](https://www.theopensourceway.org/) these topics.

## Contributing

For the 2.0 refresh, we've defined a set of roles and responsibilities around an initial, core group of contributors most familiar with the goals of the project. Beyond that, we'll evolve with the participation of new contributors and update [CONTRIBUTING.md](https://github.com/theopensourceway/guidebook/blob/master/CONTRIBUTING.md) to reflect current contribution guidelines.

At the moment:

* Project Lead Karsten Wade (@quaid) is responsible for providing vision and leadership to evolve from his original 1.0 work
* Lead Editor Brian Proffitt (@bproffitt) is responsible for leading a team of editors to work with multiple authors and content sources toward a cohesive, highly readable guide
* Lead Writer Shaun McCance (@shaunix) is responsible for working directly with writers to shape their content into the level, style, and narrative of the overall guide
* Editing team (Brian Proffitt, Karsten Wade, Shaun McCance) is responsible for polishing and integrating submissions to ensure consistency of quality and voice
* Writing team (Bryan Behrenshausen, Brian Proffitt, Karsten Wade, Shaun McCance) is responsible for composing, collecting, and otherwise curating materials included in the guidebook.

We're using AsciiDoc and using GitHub for a git-based workflow with issues for tracking each chapter and other discussions.

Want to get involved? Come talk with us on [the mailing list](https://lists.theopensourceway.org/admin/lists/contrib.theopensourceway.org/).